doc/chapters/getting-started/examples/09_setcapabilities.sh

```diff
@@ -3,6 +3,5 @@ gdbus call --system \
 --object-path /com/pelagicore/SoftwareContainerAgent \
 --method com.pelagicore.SoftwareContainerAgent.SetCapabilities \
 0 \
-"['com.pelagicore.temperatureservice.gettemperature',
-'com.pelagicore.temperatureservice.settemperature']"
+"['com.pelagicore.temperatureservice.settemperature']"
```

Comment:
It's an example only, to show how the SetCapabilities call is used inside the Getting Started chapter. The only thing I can imagine it to be good for is as an example on how to pass multiple strings in an array. I will change it  back to end the discussion ;)